prometheus_client/openmetrics/exposition.py

```diff
@@ -94,24 +111,122 @@ def generate_latest(registry):
     return ''.join(output).encode('utf-8')
 
 
-def escape_metric_name(s: str) -> str:
+def escape_metric_name(s: str, escaping: str) -> str:
     """Escapes the metric name and puts it in quotes iff the name does not
     conform to the legacy Prometheus character set.
     """
-    if _is_valid_legacy_metric_name(s):
+    print("hi", s)
```

Comment:
Debugging code here and a couple more times in this function :)

Reply:
haha oops